device_services/generic/chargepoint.md

```diff
@@ -51,6 +55,7 @@ Chargepoint service is used to represent EV chargers.
 | `sup_states`         | str_array | `["disconnected", "charging", "ready_to_charge", "finished"]` | List of possible states of the chargepoint. See [the list](#definitions) of possible values. |
 | `sup_charging_modes` | str_array | `["slow", "normal"]`                                          | Optionally supported charging modes.                                                         |
 | `sup_max_current`    | int       | `32`                                                          | Maximum current limit in `A` as set by the installer.                                        |
+| `sup_phase_modes`    | str_array | `["NL1L2L3", "NL1"]`                                          | Optionally supported phase modes. See [the list](#definitions) of possible values.           |
 | `grid_type`          | string    | `"TN"`                                                        | Grid type of the charger. Possible values are `IT`, `TT` and `TN`.                           |
```

Comment:
What about other grid types ? Not supported yet ? Shall we have "not supported" or "other" status ?

Reply:
Irrelevant for this PR. But IT, TN and TT are all grids types out there. TNS, TNC and TNCS all fall under TN type.